de-DE/Codex_der_Charaktere/06_Zauberwirken/Beschreibungen_der_Zauber/A/Attribut_verbessern.md

```diff
@@ -1,45 +1,46 @@
 #### Attribut verbessern
 <!-- markdownlint-disable link-image-reference-definitions -->
 <!-- spell-checker:words added amount avoids casting concentration damage different duration emphasis ends english false formula hour halves hours kommagetrennt mechanics minutes reaction ritual same saving school somatic special spell throw true wording wotc -->
-<!-- spell-checker:words enhance -->
+<!-- spell-checker:words enhance ability -->
 [_metadata_:spell_name]:- "Attribut verbessern"
 [_metadata_:spell_name_english]:- "Enhance Ability"
-[_metadata_:spell_school]:- "???"
-[_metadata_:spell_level]:- "???"
-[_metadata_:casting_time_amount]:- "???"
-<!-- "action", "bonus action", "reaction", "minute", "minutes", "hour", "hours" -->
-[_metadata_:casting_time_unit]:- "???"
-[_metadata_:casting_time_reaction_trigger]:- "???"
-<!-- "true" oder "false" -->
+[_metadata_:spell_school]:- "Verwandlungszauber"
+[_metadata_:spell_level]:- "2"
+[_metadata_:casting_time_amount]:- "1"
+[_metadata_:casting_time_unit]:- "Aktion"
 [_metadata_:ritual]:- "false"
-[_metadata_:range]:- "???"
-[_metadata_:target]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:components_verbal]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:components_somatic]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:components_material]:- "???"
-[_metadata_:components_material_description]:- "???"
-[_metadata_:components_material_cost]:- "???"
-<!-- "true" oder "false" -->
-[_metadata_:concentration]:- "???"
-[_metadata_:duration]:- "???"
-[_metadata_:saving_throw]:- "???"
-<!-- "halves_damage", "avoids_effect", "ends_effect", "special"; falls mehrere wahr sind, kommagetrennt einfügen -->
-[_metadata_:saving_throw_success]:- "???"
-[_metadata_:damage_formula]:- "???d???"
-[_metadata_:damage_type]:- "???"
+[_metadata_:range]:- "Berührung"
+[_metadata_:target]:- "Eine Kreatur"
+[_metadata_:components_verbal]:- "true"
+[_metadata_:components_somatic]:- "true"
+[_metadata_:components_material]:- "true"
+[_metadata_:components_material_description]:- "Fell oder Feder eines Tiers"
+[_metadata_:concentration]:- "true"
+[_metadata_:duration]:- "Konzentration, bis zu 1 Stunde"
```

Comment:
For concentration spells, in the English version I put the maximum time in this tag. We already have the fact, that it's concentration, in the tag above. So
```suggestion
[_metadata_:duration]:- "1 Stunde"
```
if we want to follow the same logic.

(The same would then have to change for _Brandmarkendes Niederstrecken_.)